tests/installation/ipxe_install.pm

```diff
@@ -109,6 +109,12 @@ boot
 END_BOOTSCRIPT
 
     diag "setting iPXE bootscript to: $bootscript";
+
+    diag "===== bootscript $autoyast =====";
```

Comment:
Maybe you should replace "bootscript" with "autoyast"?

Reply:
+1 fixed